test/verify/check-kdump

```diff
@@ -29,8 +29,8 @@ class TestKdump(MachineCase):
 
     def enableKdump(self):
         # we need to make sure that the kernel command line options include our crashkernel parameter
-        if "atomic" in self.machine.image:
-            # on atomic we have a crashkernel option already, but for auto
+        if self.machine.atomic_image or self.machine.image  in ["rhel-x"]:
+            # on atomic and rhel-x we have a crashkernel option already, but for auto
```

Comment:
can you please assert this, just incase it stops being true at some point?

alternatively, why not just 'sed -i' this?

Reply:
`sed`ing would be easier indeed, I can change this.

The test already does assert it, in line 77ff. - if. e. g. rhel-x would stop defaulting to this, then the `Service is running` assertion below would be wrong, as it would say "Service has an error" and "No memory reserved".

Reply:
I added an assertion; might be easier to debug than the failure further below, indeed.